dev/benchmarks/macrobenchmarks/test_driver/e2e_test.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(CareF): replace the following part after flutter/plugins#2859 is landed
+
```

Comment:
Is it true that we don't need this file if flutter/plugins#2859 landed? Since this PR doesn't need Timeline, I wonder if we only need a part of flutter/plugins#2859 without the Timeline support.

Reply:
My bad. This comment comes here for my first trial version. Now this file is a little different with flutter/plugins #2859 but the larger idea is similar.

Reply:
Removed the comment. See new comment in flutter/plugins#2859